src/main/resources/db/migration/all/20240607130747__create_placement_applications_automatic_table.sql

```diff
@@ -0,0 +1,14 @@
+-- create new table for Placement Applications Automatic
```

Comment:
given the entity is defined in the first commit, this migration should really be in the commit too (otherwise the prior commit fails to run). It's probably easiest to just merge the first two commits.